website/docs/community/people.md

```diff
@@ -4,4 +4,14 @@ title: People
 
 # Project Team
 
-This is content
+## Teodor Duțu
+
+![Teodor Duțu](../../static/img/people/teodor_dutu.jpg)
+
+Teodor Duțu is a contributor to the OpenEdu project (21-COP-0016).
+He is developing content for the [Operating Systems class](https://github.com/systems-cs-pub-ro/operating-systems-oer/) and the [Security Summer School](https://github.com/security-summer-school/), as well as the [D Summer School](https://dlang-upb.github.io/D-Summer-School/).
+
+Teodor is a Masters student at the University POLITEHNICA of Bucharest, Romania, enrolled in the Advanced Cybersecurity programme.
```

Comment:
```suggestion
Teodor is a Masters student at University POLITEHNICA of Bucharest, Romania, enrolled in the Advanced Cybersecurity programme.
```

Reply:
Are you sure? If it was just "University of Bucharest", there would definitely need to be a "the" before. Shouldn't it be "the POLITEHNICA University of Bucharest"? It sounds more natural to me like this.

Reply:
`the POLITEHNICA University of Bucharest` sounds most natural to me, I'm not sure what the official name in english is for UPB. Maybe @razvand can give us the truth.

Reply:
I tend to not use `the`.

Reply:
But let's settle on something, and be consistent.

Reply:
I updated my PR and I didn't use `the` there.